advanced_source/usb_semisup_learn.py

```diff
@@ -77,6 +77,8 @@
 # - ``Trainer``: a Trainer class for training and evaluating the
 # algorithm on dataset
 # 
+# Note that a CUDA-enabled backend is required for training with the ``semilearn`` package.
```

Comment:
I ran into this error when creating a `semilearn.Trainer` object:
```
---------------------------------------------------------------------------
RuntimeError                              Traceback (most recent call last)
[<ipython-input-7-a0d7949e6095>](https://localhost:8080/#) in <cell line: 1>()
----> 1 trainer = Trainer(config, algorithm)
      2 trainer.fit(train_lb_loader, train_ulb_loader, eval_loader)

2 frames
[/usr/local/lib/python3.10/dist-packages/torch/cuda/__init__.py](https://localhost:8080/#) in _lazy_init()
    296         if "CUDA_MODULE_LOADING" not in os.environ:
    297             os.environ["CUDA_MODULE_LOADING"] = "LAZY"
--> 298         torch._C._cuda_init()
    299         # Some of the queued calls may reentrantly call _lazy_init();
    300         # we need to just return without initializing in that case.

RuntimeError: Found no NVIDIA driver on your system. Please check that you have an NVIDIA GPU and installed a driver from http://www.nvidia.com/Download/index.aspx
```

Should I add instructions on how to change the runtime in Google Colab?

Reply:
You can add that info here: https://pytorch.org/tutorials/beginner/colab and add a link to it in the tutorial.